include/compartment.h

```diff
@@ -168,6 +168,10 @@ struct CompConfig
     struct CompEntryPointDef *entry_points;
     size_t entry_point_count;
     void *base_address;
+
+    char **env_ptr;
+    size_t env_ptr_sz;
+    unsigned short env_ptr_count;
```

Comment:
I think this means "how many variables are there in the environment"?

Reply:
Yes, `env_ptr_count` is the number of null terminated strings we expect to parse from `env_ptr`. Do you think I should change the name, or add a comment?

Reply:
A comment never hurts!

Reply:
Added in https://github.com/capablevms/CHERI-ELF-comp/pull/41/commits/1956ae70aca4b6053265aaae011444e673d373ee.